testproject/Assets/Tests/Manual/DeltaPositionNetworkTransform/LinearMotionHandler.cs

```diff
@@ -396,8 +396,8 @@ private void LateUpdate()
                 SetNextDirection();
             }
 
-            ClientPositionVisual.transform.position = m_ClientPosition;
-            ClientPositionVisual.transform.rotation = InLocalSpace ? transform.localRotation : transform.rotation;
+            var rotation = InLocalSpace ? transform.localRotation : transform.rotation;
```

Comment:
This still needs to use the m_ClientPosition.

Reply:
It's using `m_ClientPosition` on the next line down 😄